File: api/UrnaApi/Services/Interfaces/ICandidateService.cs
using UrnaBackend.Dtos;
using UrnaEFCore.Entities;

namespace UrnaBackend.Services.Interfaces
{
    public interface ICandidateService
    {
        public Task<bool> DeleteCandidate(int candidateId);
        public Task<bool> AddCandidate(CandidateRegisterDto candidate);
        public CandidateUrnDto GetCandidateByElectoralNumber(int electoralNumber);
    }
}

File: client/src/Services/Interfaces/ICandidateService.ts
import ICandidate from '../../Models/Interfaces/ICandidate';

interface ICandidateService {
	getCandidates(): Promise<ICandidate[]>;
	addCandidate(candidate: ICandidate): void;
	deleteCandidate(candidateId: number): void;
}

export default ICandidateService;